src/locales/en.json

```diff
@@ -704,7 +706,6 @@
 "label.dns1": "DNS 1",
 "label.dns2": "DNS 2",
 "label.domain": "Domain",
-"label.domains": "Domain",
```

Comment:
don't remember keys even  if they look redundant @davidjumani - it's used in section/titles and few other places

Reply:
It is a duplicate key, L:716

Reply:
ah okay good catch; maybe we should sort and de-dupe the list.

Reply:
Done!